### DISCUSSION TITLE
how to configure nginx proxy with subfolder

### DISCUSSION THREAD

**Discussion by Yofo-me** [AlistGo/alist#2402] [Q&A]
I wanna use such subfolder '/alist' to proxy ALIST(listen 127.0.0.1:5244),and here are my nginx config:
![image](https://user-images.githubusercontent.com/55388087/202459375-d1a6f202-c18b-458c-9856-b4f7db7797fc.png)
 
And there is screenshot when I use my domain+subfolder path to visit:
![image](https://user-images.githubusercontent.com/55388087/202459680-bda1a9f5-ed0b-4724-bacf-c2171a97d477.png)

As you can see, the web page is blank (I have configured some netdisk info when it was listening on 0.0.0.0:5244,this screenshot is token after I finished personal nginx proxy configuration ).And once I click "manage" to visit its admin panel,a 404 code return.(The web url is https://domain/@manage instead of https://domain/alist/@manage).

![image](https://user-images.githubusercontent.com/55388087/202461129-413ea551-ce1d-48dc-b200-0d20529a81a9.png)

And there is some info given in Chrome Bebug :
![image](https://user-images.githubusercontent.com/55388087/202461461-cb246dc2-0828-4356-8437-c2c7237a8912.png)

So what should I do to make it perfect?

**Comment by itsHenry35** [comment]
Did you edit this config? https://alist.nn.ci/config/configuration.html#site-url
